fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarchconnector/converter/InvoiceResponseConverter.java

```diff
@@ -0,0 +1,36 @@
+package pl.fakturogen.comarchconnector.converter;
```

Comment:
class looks fine, however I think that it should be moved to /fakturogen/comarch/connector/converter package from current /fakturogen/comarchconnector/converter